File: C#/프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cs
using System;
using System.Linq;
public class Solution {
    public int solution(int a, int b) {
        int[] aa = {int.Parse(a.ToString()+b.ToString()),int.Parse(b.ToString()+a.ToString())};
        int answer = aa.Max();
        return answer;
    }
}

File: C++/프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    string s1 = to_string(a);
    string s2 = to_string(b);
    int answer = max(stoi(s1+s2),stoi(s2+s1));
    return answer;
}